main/mafia.hs

```diff
@@ -618,8 +657,8 @@ cabal_ cmd args = do
 
 sandbox :: ProcessResult a => Argument -> [Argument] -> EitherT MafiaViolation IO a
 sandbox cmd args = do
-  sandboxDir <- getSandboxDir
-  cabal "sandbox" $ ["--sandbox", sandboxDir] <> (cmd:args)
+  _ <- initSandbox
+  cabal "sandbox" (cmd:args)
```

Comment:
It turns out that passing `--sandbox <dir>` to cabal has no effect unless you're initialising a sandbox, so I have removed it. The only configurable way of changing the sandbox to use is to set the `CABAL_SANDBOX_CONFIG` environment variable.